services/frontend-v3/src/components/competenciesCard/Competencies.jsx

```diff
@@ -11,10 +11,12 @@ class Competencies extends Component {
     let competencies = {};
     let key = 0;
 
-    data.competencies.forEach(element => {
-      competencies[key] = element.description[locale];
-      key++;
-    });
+    if (data.competencies) {
```

Comment:
everything looks good just out of curiosity why did you put in this protection? why was it not necessary before. I dont think its wrong (i actually like it) i just want to know what made you do it.

Reply:
When user "A" hide his competencies, data.competencies is null. And when data.competencies is "null", forEach throws an error.